test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalDeleteTests.cs

```diff
@@ -121,7 +130,7 @@ public async Task GivenMatchingResources_WhenDeletingConditionallyWithMultipleFl
 
             await Task.WhenAll(Enumerable.Range(1, create).Select(_ => CreateWithIdentifier(identifier)));
 
-            await GetResourceCount(identifier);
+            // await GetResourceCount(identifier);
```

Comment:
If this is commented out why not just removed it? We have the history in the repo.

Reply:
Thanks for finding this @PTaladay . I've removed that line altogether as you have stated. I saw that it was never being used to begin with.